docs/Agent/user-guide/man_page.md

```diff
@@ -333,107 +265,205 @@ pbench-register-tool-trigger, Registers the tool trigger
+thread for locally compressing the result tar balls.
 
---pbench-post STR  
-Path to the script which will be executed after tools are stopped and postprocessing is complete  
-_NOTE: --pbench-post is not compatible with --use-tool-triggers_
+`--help`  
+Show this message and exit.
 
---use-tool-triggers  
-Use tool triggers instead of normal start/stop around script;  
-_NOTE: --use-tool-triggers is not compatible with --iteration-list,--pbench-pre, or --pbench-post_
+---
 
---no-stderr-capture  
-Do not capture the stderr of the script to the result.txt file
+##### pbench-copy-results
+
+---
+
+**NAME**  
+`pbench-copy-results`, Copies result tarball to the 0.69 Pbench Server.
+
+**SYNOPSIS**  
+`pbench-copy-results [--help] --user=<user> [--controller=<controller>] [--prefix=<path>][--xz-single-threaded] [--show-server]`
+
+**DESCRIPTION**
+Push the benchmark result to the Pbench Server without removing it from the local host .We use id_rsa when pushing to a 0.69 Pbench Server.
 
---help  
+**OPTIONS**  
+`--user`  
+This option value is required if not provided by the
+'PBENCH_USER' environment variable; otherwise, a value provided
+on the command line will override any value provided by the
+environment.
+
+`--controller`  
+This option may be used to override the value
+provided by the 'PBENCH_CONTROLLER' environment variable; if
+neither value is available, the result of 'hostname -f' is used.
+(If no value is available, the command will exit with an error.)
+
+`--prefix`  
+This option allows the user to specify an optional
+directory-path hierarchy to be used when displaying the result
+files on the 0.69 Pbench Server.
+
+`--show-server`  
+This will not move any results, but resolve and
+then display the pbench server destination for results.
+
+`--xz-single-threaded`  
+This will force the use of a single
+thread for locally compressing the result tar balls.
+
+`--help`  
 Show this message and exit.
+
+---
+
+##### pbench-results-move
+
+---
+
+**NAME**  
+`pbench-results-move`, Move results tarball to the server to a 1.0 Pbench Server.
+
+**SYNOPSIS**  
+`pbench-results-move[-C][--controller][--token][--delete][--xz-single-threaded][--show-server][--help]`
+
+**DESCRIPTION**  
+Move result directories to the configured Pbench server. We use an API Key authentication token when pushing results to a 1.0 Pbench Server
+
+**OPTIONS**  
+ `-C`, `--config` PATH  
+This option is required if not provided by the `_PBENCH_AGENT_CONFIG` environment variable.
+Path to the pbench-agent configuration file.
+
+`--controller` STR  
+Override the default controller name
+
+`--token` STR  
+pbench server authentication token [required]
+
+`--delete` | `--no-delete`  
+Remove local data after successful copy [default: delete]
+
+`--xz-single-threaded`  
+Use single-threaded compression with 'xz'
+
+`--show-server` STR  
+Display information about the pbench server where the result(s) will be moved (Not implemented)
```

Comment:
It's "accurate", but seems silly to document an option that doesn't do anything, and I don't even know why we've got this. (I'm actually planning to remove it when I add `--server` and `--relay`!)